app/views/shared/_contact_panels.html.erb

```diff
@@ -119,6 +119,13 @@
 </a>
 <% end %>
 
+<a href="#" aria-hidden="true" id="js-chat-popup-mobile" class="chat-popup__container" data-dough-component="ChatPopup">
```

Comment:
It is more in keeping with our standards to avoid using id here but instead to use a data attribute. So in this case we could use data-dough-chat-popup-mobile or similar.

Reply:
I've removed the id property from this element as it's not needed anymore.